### ISSUE TITLE
In high contrast Aquatic/Desert mode, keyboard focus indicator is not visible on the data cells present in the table in windows.

### ISSUE THREAD

**Issue by msftedad** [ceolter/ag-grid#11865]
### Link to reproducible scenario

https://www.ag-grid.com/example/?theme=quartz

### Describe the bug

**Test Environment:**
OS: Windows 11 24H2 (OS Build 26100.4946)
Browser: Edge (Version 139.0.3405.102), Chrome (139.0.7258.128)
URL: https://www.ag-grid.com/example/?theme=quartz

**Pre-Requisites:**
Settings->Accessibility->Contrast themes->select Aquatic/Dessert mode.

**User Impact:**
Low vision users will have difficulty viewing the focus indicator if focus indicator is not visible on the data cells present in the table in aquatic and desert mode.

**Repro Steps:**

1. Open the URL: https://www.ag-grid.com/example/?theme=quartz in edge browser in windows.
2. Ag-grid page will open.
3. Press tab key to navigate to the table.
4. Press right/left arrow key to navigate to the data cells.
5. Verify whether in high contrast Aquatic/Desert mode, keyboard focus indicator is visible on the data cells or not.

**Actual Result:**
In high contrast Aquatic/Desert mode, keyboard focus indicator is not visible on the data cells present in the table in windows.

**Refer Attachment:**

1. In high contrast Aquatic/Desert mode, keyboard focus indicator is not visible on the data cells.png
2. In high contrast Aquatic/Desert mode, keyboard focus indicator is not visible on the data cells.mp4

https://github.com/user-attachments/assets/67469303-1b3d-4d46-b6bf-61359ddac1e9
<img width="1899" height="953" alt="Image" src="https://github.com/user-attachments/assets/abfba33b-85a1-43e6-93de-fdfb06ca8d4c" />

**Expected Result:**
In high contrast Aquatic/Desert mode, keyboard focus indicator should be visible properly on the data cells present in the table in windows.

### Version

34.0.0

### Does the issue occur for a specific framework only?

All languages

### Is the issue only observable on a specific browser?

All browsers

**Comment by msftedad** [comment]
Hi Team, Any update on this issue?

**Comment by msftedad** [comment]
Hi Team, Any update on this issue?

**Comment by msftedad** [comment]
Hi Team, please provide an update on this issue?

**Comment by kostasntolkeras** [comment]
Hello @msftedad ,

Thank you for bringing this up with us. 

The functionality you described wasn't part of the requirements when we built this feature. This is why we have added a feature request in our backlog to track this requirement and implement it in the future.

You can track it with this reference:
**AG-6163 [Accessibility] Allow the focused item to be clearly visible when using Windows high-contrast themes**

You can follow the status of feature requests, bugs and releases using our pipeline:
https://www.ag-grid.com/ag-grid-pipeline/

**Now that this requirement is recorded in our backlog, we will soon close the GitHub issue because we will not update it when this feature is added in a future version of AG Grid. The best way to get an update on this item is to sign up for AG Grid new release notifications. Once a new AG Grid version is released, look for the item reference above (AG-6163) on the changelog page to see if it's resolved in that version:
https://www.ag-grid.com/ag-grid-changelog/**

Thanks again for bringing this up with us - we appreciate your time!
